msgcntr/messageforums-app/src/webapp/js/forum.js

```diff
@@ -850,3 +850,10 @@ MFR_RBC.saveRubric = function() {
     const rubricGrading = document.getElementsByTagName("sakai-rubric-grading").item(0);
     rubricGrading && rubricGrading.release();
 };
+
+window.onload = function() {
+    var mailtos = document.querySelectorAll("[id*=createEmail1");
+    mailtos.forEach(item =>{
+        item.setAttribute('href', item.href.replaceAll('+',' '));
+    });
```

Comment:
```suggestion
    document.querySelectorAll("[id*=createEmail1").forEach(item => {
        item.setAttribute('href', item.href.replaceAll('+', ' '));
    });
```

Reply:
Actually, shouldn't you just use setAttribute and getAttribute, or do you want to do the replace on the absolute url?